composer.json

```diff
@@ -35,19 +35,19 @@
     "require": {
         "php": "^7.4|^8.0",
         "ext-json": "*",
-        "illuminate/auth": "^6|^7|^8",
-        "illuminate/contracts": "^6|^7|^8",
-        "illuminate/http": "^6|^7|^8",
-        "illuminate/support": "^6|^7|^8",
+        "illuminate/auth": "^6|^7|^8|^9",
+        "illuminate/contracts": "^6|^7|^8|^9",
+        "illuminate/http": "^6|^7|^8|^9",
+        "illuminate/support": "^6|^7|^8|^9",
         "lcobucci/jwt": "^4.0",
         "namshi/jose": "^7.0",
         "nesbot/carbon": "^1.0|^2.0"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^3",
-        "illuminate/console": "^6|^7|^8",
-        "illuminate/routing": "^6|^7|^8",
-        "mockery/mockery": "^1.4",
+        "illuminate/console": "^6|^7|^8|^9",
+        "illuminate/routing": "^6|^7|^8|^9",
+        "mockery/mockery": "^1.4.4",
```

Comment:
Any specific reason for the mockery bump?

Reply:
Tests were failing without this change on PHP Tests were failing without this change on PHP 8 & 8.1 with Laravel 9. 
https://github.com/dmason30/jwt-auth/runs/4875921133?check_suite_focus=true#step:10:22

```
1) PHPOpenSourceSaver\JWTAuth\Test\Http\ParserTest::itShouldRetrieveTheChain
ParseError: syntax error, unexpected token "|", expecting ";" or "{"

/home/runner/work/jwt-auth/jwt-auth/vendor/mockery/mockery/library/Mockery/Container.php:224
/home/runner/work/jwt-auth/jwt-auth/vendor/mockery/mockery/library/Mockery.php:117
/home/runner/work/jwt-auth/jwt-auth/tests/Http/ParserTest.php:471
```

Not sure which fix in particular.
https://github.com/mockery/mockery/blob/master/CHANGELOG.md

Reply:
Ah, because of `prefer-lowest` being part of the GHA workflow, missed that.

Thank you 🙏🏼